tests/test_formatter.py

```diff
@@ -51,3 +54,9 @@ def test_non_ascii_text_formatting(text_value, expected_py3, expected_py2):
     formatter = Formatter()
     formatted = formatter(text_value)
     assert formatted == expected
+
+
+if not six.PY2:
+    def test_can_normalize_unittest_mock_call_object():
+        formatter = Formatter()
+        print(formatter.normalize(unittest.mock.call(1, 2, 3)))
```

Comment:
I appreciate what this is doing, though wonder if this could be tested at a slightly lower level. For example, would it work to simply create an iterator here?

In the long run, like over the course of many python versions, I think tests written like that are much easier to maintain, because they don't depend on implementation details (of in this case `unittest.mock.call`). 

If it seems important to keep a test which is using unittest.mock, then perhaps both tests could be included.

Reply:
For me it was important to test this specifically for `unittest.mock.call` because I wanted to be able to snapshottest call objects (which I think is a useful use-case).

I've added a second test for a generator expression though.

Reply:
Works for me.

Sorry, one more thing. Could you add a link to the bug issue both in the code change and wit’s the tests? I’ve done this on other projects like Nock, and think over time it will help future readers.

Reply:
Yep, done :)